bigtable-grpc-interface/src/main/java/com/google/cloud/bigtable/grpc/RefreshingOAuth2CredentialsInterceptor.java

```diff
@@ -86,12 +85,12 @@ public HeaderCacheElement(IOException exception) {
 
     public CacheState getCacheState() {
       long now = System.currentTimeMillis();
-      if (now > expiresTimeMs) {
-        return CacheState.Expired;
-      } else if (now > staleTimeMs) {
+      if (now < staleTimeMs){
+        return CacheState.Good;
```

Comment:
The end result shouldn't be different.  This change is to have the most common case, the Good case, require only a single check rather than 2 checks against longs.  If "now =< staleTimeMs" then "now =< expiresTimeMs" is implicit, since staleTimeMs < expiresTimeMs.